hooks/tk-multi-publish2/maya/collector_integ.py

```diff
@@ -0,0 +1,42 @@
+import maya.cmds as cmds
+import sgtk
+from sgtk import TankError
+from sgtk.templatekey import SequenceKey
+
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class MayaSessionCollector(HookBaseClass):
```

Comment:
Please rename this class appropriately, and update the doc strings.

Reply:
the class name is changed to "MayaFocalLengthCollector" and updated doc string accordingly.